capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -0,0 +1,108 @@
+package com.google.univiz.scorecard;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * ScorecardData is a class representing a single university/college/institution from the parsed
+ * JSON data directly from the CollegeScorecard API.
+ */
+@AutoValue
+public abstract class ScorecardData {
```

Comment:
some visibility nits - since this class will only be used in this package by another class that will parse the json, it would be a good idea to make every method in this class (and the class, itself) package private. This will prevent it from leaking out past this package.

TLDR: s/public//g

Reply:
its saying im not allowed to make the class private? not sure whats happening.

Reply:
for the methods, i can remove public from it, but [i cant make them private](https://github.com/google/auto/blob/master/value/userguide/howto.md#-keep-my-accessor-methods-private). for the class, i think i can only remove public from it as well.

Reply:
See my recent comments. You'll want to use package private for things here (which means no access modifier)